main.go

```diff
@@ -85,6 +85,8 @@ func run() error {
 	defer f.Close() // nolint: golint,gosec
 	err = srv.WritePreset(f, eqPreset)
 	if err != nil {
+		_ = f.Close()
```

Comment:
I think that if something goes wrong while writing the preset it might be better to handle the error on `file.Close()` as well and in that case wrapping it and let it bubble up.